waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/client/TunnelHandler.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2016-2017 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.waggledance.client;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.jcraft.jsch.JSchException;
+import com.pastdev.jsch.tunnel.TunnelConnectionManager;
+
+import com.hotels.bdp.waggledance.api.WaggleDanceException;
+
+public class TunnelHandler {
+
+  private static final Logger LOG = LoggerFactory.getLogger(TunnelHandler.class);
+
+  private TunnelHandler() {}
+
+  public static HiveConf openTunnel(
```

Comment:
why static this makes it hard to test (need for powermock and stuff) a normal class can be injected and mocked in tests